windows/runner/resource.h

```diff
@@ -0,0 +1,16 @@
+//{{NO_DEPENDENCIES}}
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase